bundles/org.openhab.binding.mycroft/README.md

```diff
@@ -0,0 +1,161 @@
+# Mycroft Binding
+
+This binding will connect to Mycroft A.I. in order to control it or react to event by listening on the message bus.
+
+Possibilies include :
+
+- Press a button in OpenHAB to wake Mycroft without using a wake word.
```

Comment:
```suggestion
- Press a button in openHAB to wake Mycroft without using a wake word.
```

Reply:
Done, sorry. Not the first time you have to do it, and not the last time ! 😅